docs/channels/server_api/webhooks.md

```diff
@@ -103,14 +103,21 @@ The event data for this event is as follows:
 { "name": "channel_vacated", "channel": "my-channel" }
 ```
 
+### Cache miss events
```

Comment:
The title could be `Cache channels events`.

Reply:
ooh i like it, thanks!